main.cpp

```diff
@@ -113,31 +116,27 @@ struct FloatType
     FloatType& powInternal(float myNumber);
 };
 
-FloatType& FloatType::add(float myNumber)
+FloatType& FloatType::operator+=(float myNumber)
 {
     *value += myNumber;
     return *this;
 }
 
-FloatType& FloatType::subtract(float myNumber)
+FloatType& FloatType::operator-=(float myNumber)
 {
     *value -= myNumber;
     return *this;
 }
 
-FloatType& FloatType::multiply(float myNumber)
+FloatType& FloatType::operator*=(float myNumber)
 {
-    *value *= myNumber;
+    *value /= myNumber;
     return *this;
 }
 
-FloatType& FloatType::divide(float myNumber)
+FloatType& FloatType::operator/=(float myNumber)
 {
-    if (myNumber == 0.f)
-    {
-        std::cout << "You're dividing by float 0!" << std::endl;
-    }
-    *value /= myNumber; 
```

Comment:
Just because you change to operator/= doesn't mean the functionality should change.   your divide/assign function body should stay the same.

- [ ] revert to the body with the warning for your `operator/=`